contributing.md

```diff
@@ -71,10 +71,16 @@ This document provides some overall guidelines and suggestions for how to get st
   * The rebase operation may cause merge conflicts. **This is a good thing.** When you proactively resolve merge conflicts on your featuer branch, it allows our hard-working maintainers to cleanly merge your PR into development. 
-  * You will need to resolve and commit merge conflicts locally and ensure your modified code continues to function as expected. And remember ... you are the most qualified person to handle merge conflicts for the code you've written!
-* Now, push your changes to the repository on Github: `git push`  
+  * You will need to resolve and commit merge conflicts locally and ensure your modified code continues to function as expected. You can use `git add .` then `git commit -m "description"`.  And remember ... you are the most qualified person to handle merge conflicts for the code you've written!
+* Return to your branch with `git rebase --continue`
+* Check the commit log to make sure you're only pushing one commit for the branch you're working on using `git log`.
+  * Your commit should be on the very top, followed by something like `(HEAD -> your-branch-name)`. All the following commits until the next commit with a (branch-name) or (remote branch/branch name, branch name) after it should be commits you've made on your branch. If there isn't any before another branch, go on to push your changes (next bullet), else count the number of commits on your branch and see the next sub-bullet. You can exit Vim/the log screen by typing "q".
+  * To squash your commits into one commit, use `git rebase -i HEAD~[number]`, replacing "number" with the number of commits you counted above (take out the square brackets as well). You should see the number you specified of the most recent commits with the word "pick" in front of them, as well as an explanation of possible commands. You will be in the terminal's default text editor, usually Vim or Nano, in the default "edit" mode. Press "i" on your keyboard to enter "insert" mode which will allow you to make changes. Leave the topmost commit as-is (this is the most recent commit which we're squashing the other commits into), and replace the word "pick" with "f" or "fixup" for the other commits you'd like to have squashed into one. You can also edit the commit message to more accurately account for the squashed commits by replacing "pick" in the first commit with "reword" or "r". Once you're done, click "ESC" on your keyboard to leave "insert" mode and go into "edit" mode and then type "ZZ" to save and exit Vim or nano. You can double check to make sure it worked by running `git log` again.    
+* Now, push your changes to the repository on Github: `git push -f`  
```

Comment:
Mind if we also update the line below that says "It is best practice to include in the description a link to the issue the PR addresses." to now say "Please fill out the necessary information in the PR description."